free-tunes/Controllers/LoginViewController.swift

```diff
@@ -39,20 +50,7 @@ class LoginViewController: UIViewController {
         passwordInput.delegate = self
     }
 
-    func login(username: String?, password: String?) -> Bool {
-        guard let logInUsername = username, let logInPassword = password else {
-            return false
-        }
-        return checkCredentials(username: logInUsername, password: logInPassword)
-    }
-
-    func checkCredentials(username: String?, password: String?) -> Bool {
-        if username != "Admin" || password != "TestPass123" {
-            return false
-        }
-        return true
-    }
-
+// MARK: - Input Validation
     func checkTextFieldData() -> Bool {
         if usernameInput.text != "" && passwordInput.text != "" {
```

Comment:
Have you looked into isEmpty?

Reply:
I never thought of it actually. Let me have  a look at it👍